cds_ils/config.py

```diff
@@ -639,3 +641,5 @@ def _parse_env_bool(var_name, default=None):
 CDS_ILS_IMPORTER_PROVIDERS_ALLOWED_TO_DELETE_RECORDS = ["ebl", "safari"]
 
 RECORD_LEGACY_PID_TYPE = "lrecid"
+
+CDS_ILS_INDEX_LOCAL_ACCOUNTS = True
```

Comment:
Can you please add a card to the board or push a commit to the deployed envs to avoid to forget to change this var?